### PR TITLE
updating terraform files, added github actions workflows

### PR DESCRIPTION
**Pull request by yoshutch**
If we want to start using github actions for our pipeline and our ci status checks, this PR includes at least a starting point for the template to use GH actions.

It also updated the terraform to match the desired structure.

I hope this makes sense and I haven't really tested it yet. Not sure how I can test a template. I built most of this on the repo https://github.com/byu-oit/test-frontend, so you can see how it should look. I struggled with the instructions a bit, but hopefully it's descriptive enough and walks you through what it needs to.

This is a draft, until we either upload the custom actions, or create public GH actions (standard change ones) that this uses.

Thoughts?

### CODE REVIEW

**Comment by blakemorgan** [comment]
Thanks for doing this. I'd love to actually do a Zoom call and have you walk me through it sometime.

File: README.md [hunk 1]
```diff
@@ -25,38 +25,72 @@ This template includes the initial setup and scaffolding you need to create a fr
...
+        - `<APP_NAME>` - typically your repo name (keep it short if you can, there tends to be issues with longer names with some AWS resources)
+        - `<DEV_AWS_ACCT_NUM>` - the AWS account number for your dev account
+        - `<PRD_AWS_ACCT_NUM>` - the AWS account number for your prd account
+        - `<GITHUB_AWS_DEV_KEY_NAME>` - the name of the GitHub secret for dev AWS key (i.e. `byu_oit_customapps_dev_key`)
+        - `<GITHUB_AWS_DEV_SECRET_NAME>` - the name of the GitHub secret for dev AWS secret (i.e. `byu_oit_customapps_dev_secret`)
+        - `<GITHUB_AWS_PRD_KEY_NAME>` - the name of the GitHub secret for prd AWS key (i.e. `byu_oit_customapps_prd_key`)
+        - `<GITHUB_AWS_PRD_SECRET_NAME>` - the name of the GitHub secret for prd AWS secret (i.e. `byu_oit_customapps_prd_secret`)
+        - `<STD_CHANGE_TEMPLATE_ID>` - WHEN READY FOR PRODUCTION: In ServiceNow, create a new standard change template and be sure to give it an alias, put that alias here
+        - the page/site title
+        - the dependabot [config.yml](.dependabot/config.yml) file
```
Comment (snelg) [inline README.md:86]: Inconsistent path?
```
        - the dependabot [.dependabot/config.yml](.dependabot/config.yml) file
```

**Comment by blakemorgan** [comment]
Looks good. We probably don't want to merge until the standard change actions can be removed from this project. Also, maybe include some ADR's explaining what is going on in this PR and what is changing.

**Comment by yoshutch** [comment]
We're now using public github actions for starting and ending standard changes

**Pull request merged**